source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+
+<b>Technical Description</b>
+
+N/A
+
+##G0011
+
+<b>Functional Description</b>
+
+It contains the text of the error returned by the XML parser or XML validator.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0012
+
+<b>Functional Description</b>
+
+It should include the XPath location of the error. If the XPath string is to be truncated (i.e. if the length of the string is greater than 512 characters long), then the data item should not be used.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0013
+
+<b>Functional Description</b>
+
+It should be used when the error is an XML schema error concerning invalid values. The reasons for considering an attribute value invalid might be the format and/or a value for a technical code list. For such cases, the data item should contain the value of the invalid value in order to indicate which value was perceived invalid.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0014
+
+<b>Functional Description</b>
+
+Eastern longitude and Northern latitude will use the optional '+' sign.<br/>Western longitude and Southern latitude will use the '-' sign.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0015
+
+<b>Functional Description</b>
+
+In case of Incident information received via one or multiple CD180C, all the information received shall be included in the D.G. &lt;CONSIGNMENT-INCIDENT&gt;.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0016
+
+<b>Functional Description</b>
+
+&lt;CONSIGNMENT-INCIDENT-TRANSPORT EQUIPMENT.Container identification number&gt; is filled in case of incident with the new Container identification number amending initial declaration or with the existing Container identification number if DI Number of seals is greater than 0.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0017
+
+<b>Functional Description</b>
+
+'State of seals' = 1 in case that Seals are in good state (i.e. present and not damaged)<br/>'State of seals' = 0 in case that Seals are not in good state (i.e. not present or damaged).
+
```

Comment:
Grammar/style issue - Change both occurrences of `i.e.` to `that is,` (note comma).